sdk/FunctionMetadataLoaderExtension/FunctionMetadataLoaderExtension.csproj

```diff
@@ -11,7 +11,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.25" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Script.Abstractions" Version="1.0.0-preview" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Script.Abstractions" Version="1.0.3-preview" />
```

Comment:
Why 1.0.3 instead of 1.0.1?

Reply:
I just grabbed the latest available we had.

Reply:
I only see 1.0.0-preview. https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Script.Abstractions/

Reply:
The host uses [1.0.3-preview](https://github.com/Azure/azure-functions-host/blob/29fcb5a333ac89340af4d089cc1c263064393d53/src/WebJobs.Script/WebJobs.Script.csproj#L65) which is the latest (only published internally). RetryOptions do show up in [1.0.1-preview](https://github.com/Azure/azure-functions-host/commit/ccbcf7f9a492ecc661f6789ef34ab1869f4b1b15) first, but there's also an update to the RetryOptions class in [1.0.2-preview](https://github.com/Azure/azure-functions-host/commit/ac526fca803d53a7db7824e884031cb294c73d93). Don't know the progress on a public release for these, but I think we can match the host version without issues. If we only want to consume RetryOptions changes we can pick 1.0.2.